<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContributeItem</class>
 <widget class="QDialog" name="ContributeItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 18px;
color: #FFFFFF;
line-height: 22px;
font-style: normal;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res.qrc">:/images/top1.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>48</height>
      </size>
     </property>
     <widget class="CRoundLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>32</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #D197FE, stop:1 #9752ED);
border-radius: 8px;

font-family: &quot;微软雅黑&quot;;
font-size: 11px;
color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>在房间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>12</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLab">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 16px;
color: #FFFFFF;
line-height: 22px;
font-style: normal;</string>
       </property>
       <property name="text">
        <string>用户昵称占位占位占位占位占…</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 14px;
color: #A8A8A7;
line-height: 20px;
text-align: right;
font-style: normal;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 14px;
color: #ffffff;
line-height: 20px;
font-style: normal;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRoundLabel</class>
   <extends>QLabel</extends>
   <header location="global">CRoundLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
